.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -9,4 +9,4 @@ contact_links:
   about: Please learn how to report security vulnerabilities here
 - name: PSF Code of Conduct
   url: https://github.com/pypa/.github/blob/main/CODE_OF_CONDUCT.md
-  about: ❤ Be nice to other members of the community. ☮ Behave.
+  about: For all Code of Conduct related latters, please refer to this document. Take care ❤️.
```

Comment:
```suggestion
  about: For all Code of Conduct related matters, please refer to this document.
```

Reply:
oops :)

yeah, quite ok with that too :)